frontend/test/pytest/test_config_functions.py

```diff
@@ -35,7 +35,7 @@
 from catalyst.utils.toml import check_adjoint_flag, read_toml_file
 
 
-class DummyDevice(qml.QubitDevice):
+class DeviceTest(qml.QubitDevice):
```

Comment:
`DeviceTest` does not make sense as a device name, it indicates the main object is a "test" not a "device".